migrations/postgres/20240422100311_flow_configuration_events.sql

```diff
@@ -0,0 +1,26 @@
+CREATE SEQUENCE flow_configuration_event_id_seq AS BIGINT;
+
+CREATE TYPE system_flow_type AS ENUM ('gc');
+
+CREATE TABLE system_flow_configuration_events
+(
+    event_id         BIGINT PRIMARY KEY DEFAULT NEXTVAL('flow_configuration_event_id_seq'),
```

Comment:
Separate sequence/nextval are not required here, simply use BIGSERIAL type.

Reply:
I have a worry that using separate counters might add a potential problem in `save_events() -> EventID` (we have two tables), as a unique ID is expected in terms of the method contract

Do you think it's a false fear?

Reply:
1) The external world will be communicating with flow configurations via flow key. Events are implementation detail within the domain. Since this is fully encapsulated, I don't see any issue.

2) If we imagine a function, where we need to list mixed types of flows (i.e., admin view), then maybe it's an argument for single table, rather than a single sequence.

Reply:
Let it be